Client/Assets/CertError.html

```diff
@@ -45,5 +44,9 @@ <h1 id="errorTitleText">%error_title%</h1>
         advancedContent.style.display = "block";
       };
     </script>
+
+    <script>
+        %insert_css%
```

Comment:
I like the simplicity of having the WKURLSchemeHandler load the CSS and inject it.